### PR TITLE
Better explanation of implicit exception response status code

### PR DESCRIPTION
**Pull request by hvt**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | all |
| Fixed tickets | none |

### CODE REVIEW

File: reference/events.rst [hunk 1]
```diff
@@ -215,10 +215,12 @@ and set a new ``Exception`` object, or do nothing::
     response won't work. If you want to overwrite the status code (which you
     should not without a good reason), set the ``X-Status-Code`` header::
 
-        $response = Response(
+        $response = new Response(
             'Error',
-            404 // ignored,
-            array('X-Status-Code' => 200)
+            404 // this status code will be ignored,
```
Comment (javiereguiluz) [inline reference/events.rst]: the comma should go after the `404` number to avoid syntax errors (previously was also wrong of course)

Comment (hvt) [inline reference/events.rst]: haha, good catch :]

**Comment by javiereguiluz** [comment]
:+1:

**Comment by xabbuh** [comment]
:+1:

**Comment by WouterJ** [comment]
Thanks Harm! This is very easy to understand now.

**Pull request closed without merging**